openlibrary/macros/StarRatingsStats.html

```diff
@@ -12,6 +12,7 @@
       $if (stats_decimal >= 0.5) and (stats_decimal < 1):
         $:('<span class="readers-stats__star--half">★</span>')
       <span itemprop="ratingValue">$avg</span>
+      <h4>·</h4>
```

Comment:
```suggestion
      <span>·</span>
```
Let's change this to a span, and update the CSS accordingly. You can assign the span a unique class if it is helpful.

Reply:
Okey, I will fix it rigth now